docs/api/api.yml

```diff
@@ -29,15 +29,13 @@ paths:
               schema:
                 type: string
                 description: A plain text, human readable description of the service
-                example: Bag Index Service is running.
-        500:
-          $ref: '#/components/responses/Unavailable'
+                example: EASY Bag Index running v1.2.3
 
   /search?doi={DOI}:
     get:
       summary: Returns the Bag Index metadata for all items related to a digital object identifier.
       description: |
-        An item is a bag. The item fetched is specified by the DOI:
+        The item fetched is specified by the DOI. The results can contain more than one bag-info object.
```

Comment:
when the parameter-doi is not found, it returns a 200 and the result contains no bag-info objects?

Reply:
Correct, in that case it only returns: "result":[] alongside a 200